tests/test_search_issues_api.py

```diff
@@ -198,7 +198,6 @@ def test_bad_delete(self) -> None:
         assert "'query' is a required property" in res.get_json()["error"]["message"]
 
         # test for invalid column types
-        set_config("storage_deletes_enabled", 1)
```

Comment:
Would it make sense to remove these lines?
https://github.com/getsentry/snuba/blob/6fe79f0997f435c08bc4d0170ebf3ea3763c3ccd/tests/test_search_issues_api.py#L158-L164

Reply:
I had left them in just to test that the feature flagging works